app/src/main/java/org/jellyfin/androidtv/ui/preference/screen/PlaybackPreferencesScreen.kt

```diff
@@ -204,6 +203,66 @@ class PlaybackPreferencesScreen : OptionsFragment() {
 			}
 		}
 
+		category {
+			setTitle(R.string.pref_audio_directplay)
```

Comment:
I think it would be better if add a new preference screen for this category. You can look at `UserPreferencesScreen` to see how to link to one.